### PR TITLE
docs: convert from png to webp script docs

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves []

### Description

While adding some new images found that we have a script for converting images from png -> webp! 

Added some docs and moved the script to the `/scripts` directory

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- convert from png to webp script docs - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **It seems like you added some non WebP images to Eigen, please convert them to WebPs using `source images/script.sh` script **
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5f0f56accbc8b890a0b48610f7e3aa52a6deddf2
</p>

**Comment by gkartalis** [comment]
danger doesn't seem to ignore the screenshots that we add in the `/docs/screenshots` folder

**Review by MrSltun — APPROVED** [review]
Awesome!! 🔥 🌟 👏

File: docs/convert_image.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## How to import an image from figma and convert it
+
+1. Select the image in figma that you want to import in eigen.
+2. On the right side click `export`
+3. Press the `+` button 3 times for it to generate 3 files `my_image.png`, `my_image@2x.png`, `my_image@3x.png`
+4. Press the export button
+
+| Step 1,2                                    | Step 3,4                                      |
+| ------------------------------------------- | --------------------------------------------- |
+| <img src="screenshots/convert-image-1.png"> | <img src="screenshots/convert-image-2.png" /> |
```
Comment (dimatretyak) [inline docs/convert_image.md:16]: **suggestion**: record a small video, save it as `gif` and attach it here

**Review by dimatretyak — APPROVED** [review]

File: docs/convert_image.md (hunk 1 above)
Comment (gkartalis) [reply docs/convert_image.md:16]: Love this idea, much better than screenshots! Thanks @dimatretyak

**Comment by dimatretyak** [comment]
awesome, it works perfectly!

image

File: scripts/convert-png-images-to-webp.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Converts images from pngs to webps for bundle size reduction
```
Comment (pvinis) [inline scripts/convert-png-images-to-webp.sh:1]:
```suggestion
#!/usr/bin/env bash
set -euxo pipefail

# Converts images from pngs to webps for bundle size reduction
```

maybe we can add the shebang with the settings, like we do in our other scripts?

**Review by pvinis — APPROVED** [review]

**Comment by pvinis** [comment]
assigned back to you. merge when you like. i only suggested a small change, otherwise very nice!

**Pull request merged**